.github/workflows/ci-publish.yaml

```diff
@@ -0,0 +1,47 @@
+name: "Publish Docs"
+
+on:
+  workflow_run:
+    workflows: ["CI"]
+    branches: [master]
+    types: 
+      - completed
+
+jobs:
+  publishing:
+    name: Publish Documentation
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+    - name: 📥 Download stable docs with released
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: release.yaml
+        workflow_conclusion: success
+        name: stable
+        path: docs/public/
+
+    - name: 📥 Download & merge unstable docs with released
+      uses: dawidd6/action-download-artifact@v2
+      with:
+        workflow: ci.yaml
+        workflow_conclusion: success
+        name: unstable
+        path: docs/public/unstable
+
+    - name: 💾 Upload artifact
+      uses: actions/upload-artifact@v3
+      with:
+        name: head-protocol
+        path: docs/public
+        
+    # TODO enable this section before merge
+    # - name: 🚢 Publish Documentation
+    #   if: github.event_name == 'push'
+    #   uses: peaceiris/actions-gh-pages@v3
+    #   with:
+    #     github_token: ${{ secrets.GITHUB_TOKEN || github.token }}
+    #     publish_dir: docs/public
+    #     enable_jekyll: true
+    #     force_orphan: true
```

Comment:
I would uncomment the following lines

Reply:
they must be uncommented before merging to master :)